src/views/Home/components/Hero/index.view.tsx

```diff
@@ -37,6 +42,12 @@ const Hero: React.FC<HeroProps> = ({
   return (
     <>
       <div className="Hero__container">
+        <Grid className="Hero__grid" />
+        <LargeFlare className="Hero__largeFlare" />
+        <MediumFlare className="Hero__mediumFlare" />
+        <SmallFlare className="Hero__smallFlare" />
+        <Computer className="Hero__computer" />
+        <Mouse className="Hero__mouse" />
```

Comment:
I see what you did here by abstracting the three different lens flares; however, do you think we should just consolidate them apart of some `Background.svg` to be consistent with the `Milestones` and `Mission` component?

Is it better that we've abstracted them? Do they scale the same?

Reply:
good question, I decided against using a general background SVG & broke out the background elements to add more dynamics to the responsiveness for better UX. we could even animate these later on by applying CSS classes to each individual flare "component" using [react-on-visible ](https://www.npmjs.com/package/react-on-visible)

Reply:
Ohh, that makes sense! If this is the case, we might have to refactor the other few components that we finished, but it sounds  great!